### DISCUSSION TITLE
remote hosts via tcp://

### DISCUSSION THREAD

**Discussion by Famku** [lllllllillllllillll/DweebUI#83] [Q&A]
does it work with connections to multiple remote hosts via tcp://?

**Comment by lephtHanded** [comment]
I think the question here is:  (or at least the one I'm wondering)
Will dweebUI (which is great by the way) be able to manage other hosts like portainer/dokemon/dockge?

something like your home page with the host's status bars, then clicking on one of them would be able to transition to it's status bar on top and all the containers below like the current homepage.

thanks again for a great new way to manage containers!

**Comment by lllllllillllllillll** [comment]
> does it work with connections to multiple remote hosts via tcp://?

Not yet, but I plan on adding support for remote hosts.

> Will dweebUI be able to manage other hosts like portainer/dokemon/dockge?

It should be able to. Most of the application is using the Docker API, so it should work without any special setup as long it can communicate with Docker on the host.

>something like your home page with the host's status bars, then clicking on one of them would be able to transition to it's status bar on top 
and all the containers below like the current homepage.

Yeah. I think we're picturing the same thing on that. There are actually some hidden buttons in there right now.

![navbar](https://github.com/lllllllillllllillll/DweebUI/assets/147879489/2f4e4ec9-0f99-4b23-a904-a3c56b099676)

>which is great by the way

Thanks! I'm glad other people are actually using it.
